uwsgi.h

```diff
@@ -1362,7 +1362,8 @@ struct wsgi_request {
 	char *appid;
 	uint16_t appid_len;
 
-	//this is big enough to contain sockaddr_in
+	// This structure should not be used any more
+	// in favor of the union client_addr at the end
 	struct sockaddr_un c_addr;
```

Comment:
Would it make sense to rename this so any code trying to build against it would fail loudly?

Reply:
I don't know if anyone cares about symbol compatibility or not; if not, then that sounds like a good idea.